### PR TITLE
added participants page for Raul w a photo

### LINKED ISSUE #27: Please add Raul to list
Please add Raul to the list of workshop participants 😃

### PR DESCRIPTION
**Pull request by raulrd3**
Fixes #27

### CODE REVIEW

File: pages/participants/raul.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <Layout>
+      <About
+        name="Raul Rodriguez III"
+        imageUrl="http://www.fillmurray.com/500/300"
+        description="Hey, there! My name is Raul and I'm taking this workshop"
+      />
+    </Layout>
+  );
+}
+
```
Comment (chrissygonzalez) [inline pages/participants/raul.js:15]: this is awesome!! :sparkles:

Comment (chandativey) [reply pages/participants/raul.js:15]: learning so much!! 😄

**Comment by egiurleo** [comment]
Thanks for the contribution, @raulrd3! Could you remove `package-lock.json` from this PR? You can do that by running the following commands from your `codeland-2020` folder:

```
git checkout main
git pull upstream main
git checkout raulrd3
git rebase main
git rm --cached package-lock.json

git add package-lock.json
git commit -m "Removed package-lock.json"
git push
```

**Comment by raulrd3** [comment]
@egiurleo not sure if that worked as I got an error message  after copy pasting the commands in the terminal and then tried pushing again and it went though 🤷‍♂️

**Pull request closed without merging**